app/keepers/keepers.go

```diff
@@ -103,384 +103,401 @@ type appKeepers struct {
 	SuperfluidKeeper     *superfluidkeeper.Keeper
 	GovKeeper            *govkeeper.Keeper
 	WasmKeeper           *wasm.Keeper
-}
-
-func (app *OsmosisApp) InitSpecialKeepers(
-	skipUpgradeHeights map[int64]bool,
-	homePath string,
-	invCheckPeriod uint,
-) {
-	appCodec := app.appCodec
-	bApp := app.BaseApp
-	cdc := app.cdc
-	keys := app.keys
-	tkeys := app.tkeys
-	memKeys := app.memKeys
-
-	paramsKeeper := app.initParamsKeeper(appCodec, cdc, keys[paramstypes.StoreKey], tkeys[paramstypes.TStoreKey])
-	app.ParamsKeeper = &paramsKeeper
+	// transfer module
+	TransferModule transfer.AppModule
 
-	// set the BaseApp's parameter store
-	bApp.SetParamStore(app.ParamsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramskeeper.ConsensusParamsKeyTable()))
+	// keys to access the substores
+	keys    map[string]*sdk.KVStoreKey
+	tkeys   map[string]*sdk.TransientStoreKey
+	memKeys map[string]*sdk.MemoryStoreKey
```

Comment:
Oh interesting, right we do still need the map stored. Good call on moving it to this package!